app/src/main/cpp/logger-android.cc

```diff
@@ -0,0 +1,54 @@
+#include "pch.h"
+
+#include "logger.h"
+
+namespace zen::display_system::oculus {
+
+namespace {
+
+class AndroidLogger : public ILogger {
```

Comment:
Why did you create the class (and interface)? The function like zen seems enough.

Reply:
For the sake of code consistency, I don't want to force the developer to choose between using a function or a class in this project, which is written in C++. We want to use classes for the basics, and functions only for the parts where there is no room for confusion.
As for the use of interfaces, for this project I will try to write the interface in the header and the implementation in the .cc file.
This technique is often used in object-oriented languages and may not be necessary for this project, but it is likely to be a necessary technique when increasing device support in the future, so I would like to accumulate knowledge here.